.github/actions/install-deps/action.yml

```diff
@@ -0,0 +1,98 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Install dependencies
+description: Install system dependencies needed for OpenTitan
+
+inputs:
+  verilator-version:
+    description: Verilator version to install
+    required: true
+    default: '4.210'
+  verilator-path:
+    description: Path at which to install Veriltator
+    required: true
+    default: /tmp/verilator
```

Comment:
Maybe `sudo mkdir -p /tools && sudo chown $USER: /tools` would allow the normal path while allowing caching?

Reply:
Changed back to `/tools/verilator` now the cache is removed